web/book/src/tutorial/aggregation.md

```diff
@@ -2,14 +2,14 @@
 
 A key feature of analytics is reducing many values down to some summary. This
 act is called "aggregation" and always includes a function &mdash; for example,
-`average` or `sum` &mdash; that reduces the table to a single row.
+`average` or `sum` &mdash; that reduces values in the table to a single row.
 
 ### `aggregate` transform
 
 The `aggregate` transform takes a tuple to create one or more new columns that
 "distill down" data from all the rows.
 
-```
+```prql no-eval
```

Comment:
Why are these all `no-eval`?

Reply:
My understanding the `prql` gives the syntax coloring; the `no-eval` doesn't create the corresponding SQL block.

Reply:
Yes re adding `prql`!

Do we want the SQL block though? Or this is intended for folks who don't know SQL? (Though in that case we are going to have to make it more basic I think? We're talking about relations & tuples, we might need to analogize to a spreadsheet...)

Thoughts @aljazerzen ?

Reply:
I prefer to encourage readers to use the Playground. It has the double benefit of showing the SQL, and also showing the result of the query with one easy copy/paste.